go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gardener/gardener-extension-shoot-lakom-service
 
-go 1.24.0
+go 1.24.3
```

Comment:
Not very nice but there is nothing we can do right now, it is coming from https://github.com/buildkite/shellwords through sigsotre/cosign,